javacpp failing with android ndk r20

```
i686-linux-android/bin/ld: error: cannot find -lc++
```

according to https://bugreports.qt.io/browse/QTBUG-76293 a workaround is to add `-nostdlib++`. i'm not sure if we can add this to `platform.compiler.default` unconditionally or if it will affect older ndk releases adversely.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,4 +1,6 @@
 
+ * Fix Android build properties for NDK r20b ([pull #357](https://github.com/bytedeco/javacpp/pull/357))
+
 ### November 5, 2019 version 1.5.2
  * Provide `ByteIndexer` with value getters and setters for unsigned `byte` or `short`, `half`, `bfloat16`, and `boolean` types as well
  * Introduce `PointerScope.extend()` to prevent deallocation on the next call to `close()`

--- a/src/main/resources/org/bytedeco/javacpp/properties/android-arm-clang.properties
+++ b/src/main/resources/org/bytedeco/javacpp/properties/android-arm-clang.properties
@@ -16,7 +16,7 @@ platform.compiler.fastfpu=-mfpu=neon -ffast-math -Wl,--fix-cortex-a8
 platform.compiler.nodeprecated=-Wno-deprecated-declarations
 platform.compiler.noexceptions=-fno-exceptions -fno-rtti
 platform.compiler.nowarnings=-w
-platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target armv7-none-linux-androideabi -march=armv7-a -mfloat-abi=softfp -Wall -static-libstdc++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
+platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target armv7-none-linux-androideabi -march=armv7-a -mfloat-abi=softfp -Wall -nostdlib++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
 platform.compiler.output2=-Wl,-soname,
 platform.linkpath.prefix=-L
 platform.linkpath=sources/cxx-stl/llvm-libc++/libs/armeabi-v7a/

--- a/src/main/resources/org/bytedeco/javacpp/properties/android-arm64-clang.properties
+++ b/src/main/resources/org/bytedeco/javacpp/properties/android-arm64-clang.properties
@@ -16,7 +16,7 @@ platform.compiler.fastfpu=-ffast-math
 platform.compiler.nodeprecated=-Wno-deprecated-declarations
 platform.compiler.noexceptions=-fno-exceptions -fno-rtti
 platform.compiler.nowarnings=-w
-platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target aarch64-none-linux-android -march=armv8-a -Wall -static-libstdc++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
+platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target aarch64-none-linux-android -march=armv8-a -Wall -nostdlib++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
 platform.compiler.output2=-Wl,-soname,
 platform.linkpath.prefix=-L
 platform.linkpath=sources/cxx-stl/llvm-libc++/libs/arm64-v8a/

--- a/src/main/resources/org/bytedeco/javacpp/properties/android-x86-clang.properties
+++ b/src/main/resources/org/bytedeco/javacpp/properties/android-x86-clang.properties
@@ -16,7 +16,7 @@ platform.compiler.fastfpu=-msse3 -ffast-math -mfpmath=sse
 platform.compiler.nodeprecated=-Wno-deprecated-declarations
 platform.compiler.noexceptions=-fno-exceptions -fno-rtti
 platform.compiler.nowarnings=-w
-platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target i686-none-linux-android -march=i686 -mtune=atom -Wall -static-libstdc++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
+platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target i686-none-linux-android -march=i686 -mtune=atom -Wall -nostdlib++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
 platform.compiler.output2=-Wl,-soname,
 platform.linkpath.prefix=-L
 platform.linkpath=sources/cxx-stl/llvm-libc++/libs/x86/

--- a/src/main/resources/org/bytedeco/javacpp/properties/android-x86_64-clang.properties
+++ b/src/main/resources/org/bytedeco/javacpp/properties/android-x86_64-clang.properties
@@ -16,7 +16,7 @@ platform.compiler.fastfpu=-msse3 -ffast-math
 platform.compiler.nodeprecated=-Wno-deprecated-declarations
 platform.compiler.noexceptions=-fno-exceptions -fno-rtti
 platform.compiler.nowarnings=-w
-platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target x86_64-none-linux-android -march=x86-64 -mtune=atom -Wall -static-libstdc++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
+platform.compiler.output=-Wl,-rpath,lib/ -D__ANDROID_API__=21 -DANDROID -ffunction-sections -funwind-tables -fstack-protector-strong -target x86_64-none-linux-android -march=x86-64 -mtune=atom -Wall -nostdlib++ -fPIC -shared -Wl,--no-undefined -z text -o\u0020
 platform.compiler.output2=-Wl,-soname,
 platform.linkpath.prefix=-L
 platform.linkpath=sources/cxx-stl/llvm-libc++/libs/x86_64/

